api/commands/build.ts

```diff
@@ -82,12 +82,12 @@ export class Builder {
     const tasks = t.disabled
       ? emptyTasks
       : emptyTasks.concat(
-          t.incrementalQuery
-            ? (t.incPreOps || []).map(pre => ({ statement: pre }))
+          t.type === "incremental" && t.incrementalPreOps
```

Comment:
i think this is still incorrect. we should only use `incrementalPreOps` in the same circumstances where we'll run the incremental query.

see https://source.cloud.google.com/tada-analytics/github_dataform-co_dataform/+/master:core/adapters/bigquery.ts;l=29

this logic determines whether to run the plain `query`, or to run the `incrementalQuery` (and it also has fallback logic to the old-incremental style in that case). i would suggest pushing preops/postops down into `publishTasks`, so that the logic can be shared.

Reply:
Have migrated them to individual adapters